framework/docker/ibc/types.go

```diff
@@ -0,0 +1,37 @@
+package ibc
```

Comment:
these are the types that we use in the tests to configure how hermes should create channels/ connections etc